src/SFA.DAS.EmployerFinance.Web/appsettings.Development.json

```diff
@@ -1,4 +1,7 @@
 {
+  "ConnectionStrings": {   
+    "RedisLogging": ""
```

Comment:
`localhost:6379` can/should be used for the development `RedisLogging` value.

Reply:
done